src/coreComponents/mainInterface/ProblemManager.cpp

```diff
@@ -511,17 +511,22 @@ void ProblemManager::generateMesh()
 
       GeometricObjectManager & geometricObjects = this->getGroup< GeometricObjectManager >( groupKeys.geometricObjectManager );
 
+      // Nodes
+      nodeManager.setNodesInformation( cellBlockManager );
       MeshUtilities::generateNodesets( geometricObjects, nodeManager );
```

Comment:
```suggestion
      MeshUtilities::generateNodeSets( geometricObjects, nodeManager );
```

Reply:
Do you mean that I should correct the typo?
(I'm OK, just to be sure to fully understand your patch/comment)

Reply:
ah, I guess you ll have to change the name of the function as well. Just coz everywhere else we use `nodeSets`. Not a particularly useful comment :)

Reply:
It's OK, let's take all the improvements everywhere!